indico/web/client/js/ckeditor.js

```diff
@@ -0,0 +1,108 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2022 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+export const getConfig = ({simple = true, images = false} = {}) => ({
+  language: 'en_GB',
```

Comment:
Can't we take this from the currently used locale instead?

Reply:
I just checked and we currently don't set a locale either (and nobody ever complained) so fine for me.